.github/workflows/create-quantic-package.yml

```diff
@@ -57,6 +57,7 @@ jobs:
           npm run setup
           cd packages/quantic
           npm run build
```

Comment:
from package.json:
```
    "build": "npm run copy:staticresources",
```
seems like we're being redundant here. What is this intended to fix?

Reply:
Didn't notice that that was already done with npm run build... It was intended to fix the error from the last run which was said something like `Referenced file missing from this directory: ...../staticresources/coveoheadless`. You can see the full error in the screenshot from the PR description. Not sure what causes the error then

Reply:
Looks like the copy command is indeed failing
```
Begin copy.
(node:2119) UnhandledPromiseRejectionWarning: [object Array]
(Use `node --trace-warnings ...` to show where the warning was created)
(node:2119) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:2119) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```